packages/elements/src/item/index.ts

```diff
@@ -187,6 +188,16 @@ export class Item extends ControlElement {
     }
   }
 
+  /**
+   * Called after the component is first rendered
+   * @param changedProperties Properties which have changed
+   * @returns {void}
+   */
+  protected firstUpdated (changedProperties: PropertyValues): void {
+    super.firstUpdated(changedProperties);
+    registerOverflowTooltip(this, () => this.textContent, () => this.isTruncated);
```

Comment:
And why `isTruncated` is a public method? I think we can remove the method and use `isElementOverflown`, similar to text-field

Reply:
`isTruncated` is a read-only property in item since v4. I can use `isElementOverflown` instead, but I'm not sure if it's ok to remove `isTruncated`?

Reply:
remove it

Reply:
Cool! updated! 😆